src/components/EmailSignup/Wrapper.tsx

```diff
@@ -1,11 +1,17 @@
-import { Box, Flex, BoxProps } from '@chakra-ui/react';
+import { Box, Flex, FlexProps } from '@chakra-ui/react';
 import { ReactNode } from 'react';
 
-interface WrapperProps extends BoxProps {
+interface WrapperProps extends FlexProps {
```

Comment:
hmm I guess this should still extend  BoxProps since we're spreading it on a box?

Reply:
Not sure if the types are different though

Reply:
Minor differences.